ApollosProject/apollos-prototype
# Include items from Channels that don't require approval

## DESCRIPTION

### What does this PR do, or why is it needed?

In Rock, some `ContentChannels` don't require their items to be approved. The following query update ensures that we include items from ContentChannels that don't require approval when using the live filter. 

### What design trade-offs/decisions were made?
n/a

### How do I test this PR?
1. Add `ROCK_SHOW_INACTIVE_CONTENT=false` to your api .env file
2. Visit the home feed.
3. You should see "Spur One Another" as the featured content. This item won't show up in master. 

### What automated tests did you write?

Updated existing snapshots.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
